rokwire.yaml

```diff
@@ -333,6 +335,108 @@ paths:
                   $ref: '#/components/schemas/Notification'
         401:
           description: Unauthorized
+  /events:
+    get:
```

Comment:
Would a GET operation on /events/{event_id} be helpful to retrieve detailed information about an event, assuming GET /events operation only return minimum information about events such as names and dates?

Reply:
The endpoint can take event_id inside the query string.
The get endpoint wants to take a query string:
https://opensource.ncsa.illinois.edu/confluence/display/SCCI/Backend+-+Building+Blocks